src/main/java/com/google/sps/data/IAMBindingDatabaseEntry.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
```

Comment:
is this the appropriate package name? would the three of you want to come up with something fun and related to the project title?

Reply:
we've renamed it to com.google.impactdashboard